inventory/host_vars/all

```diff
@@ -0,0 +1,4 @@
+ansible_connection: ssh
+ansible_user: centos
+#ansible_ssh_private_key_file: "{{ lookup('env','HOME') }}/.ssh/{{ key_name }}.pem"
+ansible_ssh_private_key_file: "{{ lookup('env','HOME') }}/.ssh/tng-infra.pem"
```

Comment:
I think currently Jenkins is using .ssh/id_rsa and not .ssh/tng-infra.pem. Can we change it?